Microsoft.NET.Build.Containers/Resources/Strings.resx

```diff
@@ -121,25 +121,25 @@
     <value>Request to Amazon Elastic Container Registry failed prematurely. This is often caused when the target repository does not exist in the registry.</value>
   </data>
   <data name="BlobUploadFailed" xml:space="preserve">
-    <value>Failed to upload blob to {0}; received {1} with detail {2}</value>
+    <value>Failed to upload blob to {0}; received {1} with detail {2}.</value>
   </data>
   <data name="CouldntDeserializeJsonToken" xml:space="preserve">
-    <value>Could not deserialize token from JSON</value>
+    <value>Could not deserialize token from JSON.</value>
   </data>
   <data name="DockerInfoFailed" xml:space="preserve">
-    <value>Failed to get docker info({0})\n{1}</value>
+    <value>Failed to get docker info({0})\n{1}\n{2}.</value>
```

Comment:
if stdout is multiline, then period is not needed. 
stdout should have a period if formatted correctly.